src/plugins/janus_sip.c

```diff
@@ -7392,6 +7419,56 @@ gpointer janus_sip_sofia_thread(gpointer user_data) {
 	return NULL;
 }
 
+/* check peer rtp has rfc2833 and push event */
```

Comment:
Code style: please start comments with a capitalized letter. Besides, both "rtp" and rfc" should be all caps. I see other comments that should be fixed the same way.

Reply:
I fixed it.